recipes-ofdpa/ofdpa/ofdpa_3.0.5.0-EA5.bb

```diff
@@ -6,7 +6,7 @@ PV = "3.0.5.0-EA5+sdk-${SDK_VERSION}+gitAUTOINC+${@'${SRCREV_ofdpa}'[:10]}_${@'$
 
 PR = "r13"
 SDK_VERSION = "6.5.24"
-SRCREV_ofdpa = "65302857188a55cf23fb8bbb72dc592baaa9c5c6"
```

Comment:
this doesn't match with the previous version internally, you must have dropped commits before this one.

Reply:
> 
I don't know which commits are missing, I already checked manually once

Reply:
you are missing "ofdpa: do not disable platforms manually".